In [1]:
import json

with open('codigo_civil.json') as json_file:
    data = json.load(json_file)

In [2]:
import pandas as pd

pd.read_json('codigo_civil.json')

introducao  \
LIVRO I             \nPresidência da República\nCasa Civil\nSubche...   
LIVRO II            \nPresidência da República\nCasa Civil\nSubche...   
LIVRO III           \nPresidência da República\nCasa Civil\nSubche...   
LIVRO IV            \nPresidência da República\nCasa Civil\nSubche...   
LIVRO V             \nPresidência da República\nCasa Civil\nSubche...   
LIVRO COMPLEMENTAR  \nPresidência da República\nCasa Civil\nSubche...   

                                                           assinatura  \
LIVRO I             Brasília, 10 de janeiro de 2002; 181 o da Inde...   
LIVRO II            Brasília, 10 de janeiro de 2002; 181 o da Inde...   
LIVRO III           Brasília, 10 de janeiro de 2002; 181 o da Inde...   
LIVRO IV            Brasília, 10 de janeiro de 2002; 181 o da Inde...   
LIVRO V             Brasília, 10 de janeiro de 2002; 181 o da Inde...   
LIVRO COMPLEMENTAR  Brasília, 10 de janeiro de 2002; 181 o da Inde...   

                                                               indice  \
LIVRO I             ÍNDICE\n\nP A R T E G E R A L\nLIVRO I DAS PES...   
LIVRO II            ÍNDICE\n\nP A R T E G E R A L\nLIVRO I DAS PES...   
LIVRO III           ÍNDICE\n\nP A R T E G E R A L\nLIVRO I DAS PES...   
LIVRO IV            ÍNDICE\n\nP A R T E G E R A L\nLIVRO I DAS PES...   
LIVRO V             ÍNDICE\n\nP A R T E G E R A L\nLIVRO I DAS PES...   
LIVRO COMPLEMENTAR  ÍNDICE\n\nP A R T E G E R A L\nLIVRO I DAS PES...   

                                                          parte geral  \
LIVRO I             {'nome': 'DAS PESSOAS', 'TÍTULO I': {'header':...   
LIVRO II            {'nome': 'DOS BENS', 'TÍTULO I': {'header': 'T...   
LIVRO III           {'nome': 'Dos Fatos Jurídicos', 'TÍTULO I': {'...   
LIVRO IV                                                          NaN   
LIVRO V                                                           NaN   
LIVRO COMPLEMENTAR                                                NaN   

                                                       parte especial  
LIVRO I             {'nome': 'DO DIREITO DAS OBRIGAÇÕES', 'TÍTULO ...  
LIVRO II            {'nome': 'Do Direito de Empresa', 'TÍTULO I': ...  
LIVRO III           {'nome': 'Do Direito das Coisas', 'TÍTULO I': ...  
LIVRO IV            {'nome': 'Do Direito de Família', 'TÍTULO I': ...  
LIVRO V             {'nome': 'Do Direito das Sucessões', 'TÍTULO I...  
LIVRO COMPLEMENTAR  {'header': 'LIVRO COMPLEMENTAR
DAS Disposições...

In [3]:
v_tit_keys = list(data['parte especial']['LIVRO V'].keys())
v_tit_keys

['nome', 'TÍTULO I', 'TÍTULO II', 'TÍTULO III', 'TÍTULO IV']

In [4]:
v_tit_keys = v_tit_keys[1:]
v_tit_keys

['TÍTULO I', 'TÍTULO II', 'TÍTULO III', 'TÍTULO IV']

In [5]:
for key in v_tit_keys:
    print(data['parte especial']['LIVRO V'][key].keys())

dict_keys(['header', 'artigos', 'CAPÍTULO I', 'CAPÍTULO II', 'CAPÍTULO III', 'CAPÍTULO IV', 'CAPÍTULO V', 'CAPÍTULO VI', 'CAPÍTULO VII'])
dict_keys(['header', 'artigos', 'CAPÍTULO I', 'CAPÍTULO II', 'CAPÍTULO III'])
dict_keys(['header', 'artigos', 'CAPÍTULO I', 'CAPÍTULO II', 'CAPÍTULO III', 'CAPÍTULO IV', 'CAPÍTULO V', 'CAPÍTULO VI', 'CAPÍTULO VII', 'CAPÍTULO VIII', 'CAPÍTULO IX', 'CAPÍTULO X', 'CAPÍTULO XI', 'CAPÍTULO XII', 'CAPÍTULO XIII', 'CAPÍTULO XIV'])
dict_keys(['header', 'artigos', 'CAPÍTULO I', 'CAPÍTULO II', 'CAPÍTULO III', 'CAPÍTULO IV', 'CAPÍTULO V', 'CAPÍTULO VI', 'CAPÍTULO VII'])


In [6]:
topic = ''.join(data['parte especial']['LIVRO V']['TÍTULO I']['CAPÍTULO I']['artigos'])
topic

'Art. 1.784. Aberta a sucessão, a herança transmite-se, desde logo, aos herdeiros legítimos e testamentários.\n\n  Art. 1.785. A sucessão abre-se no lugar do último domicílio do falecido.\n\n  Art. 1.786. A sucessão dá-se por lei ou por disposição de última vontade.\n\n  Art. 1.787. Regula a sucessão e a legitimação para suceder a lei vigente ao tempo da abertura daquela.\n\n  Art. 1.788. Morrendo a pessoa sem testamento, transmite a herança aos herdeiros legítimos; o mesmo ocorrerá quanto aos bens que não forem compreendidos no testamento; e subsiste a sucessão legítima se o testamento caducar, ou for julgado nulo.\n\n  Art. 1.789. Havendo herdeiros necessários, o testador só poderá dispor da metade da herança.\n\n  Art. 1.790. A companheira ou o companheiro participará da sucessão do outro, quanto aos bens adquiridos onerosamente na vigência da união estável, nas condições seguintes: (Vide Recurso Extraordinário nº 646.721) (Vide Recurso Extraordinário nº 878.694)\n\nI - se concorrer

# Testando CrewAI Multiagentes

In [7]:
# Install the mains crewAI package
!pip install crewai

# Install the main crewAI package and the tools package
# that includes a series of helpful tools for your agents
!pip install 'crewai[tools]'

In [8]:
# from getpass import getpass
# GROQ_API = getpass("API groq")

KeyboardInterrupt: Interrupted by user

In [19]:
import os
from getpass import getpass
# os.environ["GROQ_API_KEY"] = 'GROQ_API'
# os.environ["OPENAI_API_KEY"] = getpass("API OpenAI")
os.environ["SERPER_API_KEY"] = getpass("API Serper")

API Serper··········


In [9]:
!pip install langchain-groq

In [20]:
from langchain_groq import ChatGroq

llama3 = ChatGroq(temperature=0, model_name="llama3-70b-8192", api_key='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')

llama3.invoke("What is the capital of France?")

AIMessage(content='The capital of France is Paris.', response_metadata={'token_usage': {'completion_tokens': 7, 'prompt_tokens': 17, 'total_tokens': 24, 'completion_time': 0.01880575, 'prompt_time': 0.005711045, 'queue_time': None, 'total_time': 0.024516795}, 'model_name': 'llama3-70b-8192', 'system_fingerprint': 'fp_abd29e8833', 'finish_reason': 'stop', 'logprobs': None}, id='run-c9c49d5e-0607-41ea-a150-847191db1503-0')

In [21]:
from crewai import Agent
from crewai_tools import SerperDevTool
search_tool = SerperDevTool()

# Creating a senior researcher agent with memory and verbose mode
researcher = Agent(
    llm=llama3,
    role='Brazilian Lawyer Researcher',
    goal='Understand situations where the {topic} can be applied to',
    verbose=True,
    memory=True,
    backstory=(
    "Being a dedicated lawyer in Brazil, you look for relevant"
    "informations about the Código Civil Brasileiro and share this"
    "knowledge to your clients."
    ),
    tools=[search_tool],
    allow_delegation=True
)

# Creating a writer agent with custom tools and delegation capability
writer = Agent(
    llm=llama3,
    role='Writer',
    goal='Imagine and narrate real world stories about {topic}',
    verbose=True,
    memory=True,
    backstory=(
    "With a flair for simplifying complex topics, you craft"
    "realistic narratives that can occur in Brazilians' daily life."
    ),
    tools=[search_tool],
allow_delegation=False
)

In [22]:
from crewai import Task

# Research task
research_task = Task(
  description=(
    "Identify informations about the {topic}."
    "Focus on identifying newly information, and if the topic is updated."
    "Also look for correlated laws. Your final report should clearly articulate"
    "the key points, history of the law, and correlated laws."
  ),
  expected_output='A comprehensive 3 paragraphs long report writen in Brazilian Portuguese.',
  tools=[search_tool],
  agent=researcher,
)

# Writing task with language model configuration
write_task = Task(
  description=(
    "Compose an true story where {topic} can be applied."
    "Focus on the real situations."
    "This article should be easy to understand and give a problem that could be"
    "solved by the law pointed in topic"
  ),
  expected_output='A 4 paragraph article on {topic} writen in Brazilian Portuguese.',
  tools=[search_tool],
  agent=writer,
  async_execution=False,
  #output_file='new-blog-post.md'  # Example of output customization
)

In [23]:
from crewai import Crew, Process

# Forming the tech-focused crew with some enhanced configurations
crew = Crew(
  agents=[researcher, writer],
  tasks=[research_task, write_task],
  process=Process.sequential,  # Optional: Sequential task execution is default
  memory=False, # memory default True needs openai
  cache=True,
  max_rpm=100,
  share_crew=True
)

In [24]:
# Starting the task execution process with enhanced feedback
result = crew.kickoff(inputs={'topic': topic})
print(result)



> Entering new CrewAgentExecutor chain...
Thought: I need to gather information about the Art. 1.784 to 1.790 of the Brazilian Civil Code.

Action: Search the internet
Action Input: {"search_query": "Art. 1.784 to 1.790 of the Brazilian Civil Code"} 


Search results: Title: Law No. 10.406 ​​of January 10, 2002 (Civil Code), Brazil, WIPO Lex
Link: https://www.wipo.int/wipolex/en/legislation/details/9615
Snippet: Brazil - Year of Version: 2010 - Adopted: December 7, 2002 - IP-related Laws - Other.
---
Title: [DOC] Civil Code of Brazil - Seafarers Rights International
Link: https://seafarersrights.org/wp-content/uploads/2018/03/BRA_CODE-OF-CONDUCT_BRAZILIAN-BAR-CODE-OF-ETHICS_1995_PRT.docx
Snippet: cuted in Brazil, subject to (mediante) the conditions which. the Brazilian law may prescribe. Art. 17. The laws, acts and judgments of other coun-. tries, as ...
---
Title: [PDF] the reach of the parter's sucession rights with stf's jugdment of the ...
Link: http://revista.fumec.br/index.php

RateLimitError: Error code: 429 - {'error': {'message': 'Rate limit reached for model `llama3-70b-8192` in organization `org_01hwnqgnerfaa9xfxtytmhvtbx` on tokens per minute (TPM): Limit 6000, Used 9521, Requested ~2875. Please try again in 1m3.961s. Visit https://console.groq.com/docs/rate-limits for more information.', 'type': 'tokens', 'code': 'rate_limit_exceeded'}}

# Testando criação de perguntas sem Agentes

In [27]:
system_check_message = "You should answer in JSON format with two attribute: 'ans' and 'connection_ok'."

messages=[
    {
        "role": "system",
        "content": system_check_message
    },
    {
        "role": "user",
        "content": "Olá! Você está ai?"
    }
  ]

response = llama3.invoke(messages)
response.content

'{ "ans": "Sim, estou aqui!", "connection_ok": true }'

In [43]:
sys_story_message = """
Consider:
- a given context about a part of the Código Civil Brasileiro.
- some articles in this part.
- and some real example questions
Create a real world story for a Brazilian daily life contain a problem
that could be answer by the law pointed in context and articles.
The story should be easy to understand and give a problem that could be
solved by the law pointed in context.
Also justify your answer.
Answer in Brazilian Portuguese.
""".strip()

user_story_prompt = """
- "context": "{context}",
- "articles": "{articles}",
- "examples": "{examples}"
""".strip()

In [44]:
exemplos = '\n'.join([
    "Para destinar a herança é preciso ter testamento",
    "Qual a diferença do divórcio consensual e o divórcio litigioso?",
    "Posso alterar o regime de bens que escolhi para o meu casamento para qualquer outro, ou há alguma restrição?",
    "Filhos fora do casamento podem receber herança?"
])

In [45]:
context = ''. join(
    [data['introducao'],
     data['parte especial']['LIVRO V']['nome'],
     data['parte especial']['LIVRO V']['TÍTULO I']['header'],
     data['parte especial']['LIVRO V']['TÍTULO I']['CAPÍTULO I']['header']]).replace('\n', ' ')
context

' Presidência da República Casa Civil Subchefia para Assuntos Jurídicos  LEI Nº 10.406, DE 10 DE JANEIRO DE 2002  Texto compilado  ÍNDICE  Vigência  Lei de Introdução às normas do Direito Brasileiro  (Vide Lei nº 14.195, de 2021)  Vide Lei nº 14.451, de 2022   Vigência  Institui o Código Civil.  O PRESIDENTE DA REPÚBLICA Faço saber que o Congresso Nacional decreta e eu sanciono a seguinte Lei:  Do Direito das SucessõesTÍTULO I Da Sucessão em Geral  CAPÍTULO I Disposições Gerais    '

In [46]:
messages=[
    {
        "role": "system",
        "content": sys_story_message
    },
    {
        "role": "user",
        "content": user_story_prompt.format(context=context,
                                            articles=topic,
                                            examples=exemplos)
    }
  ]

response = llama3.invoke(messages)
response.content

'História:\n\nMaria e João estavam em uma união estável há 10 anos. Eles compraram uma casa juntos e tinham uma vida financeira compartilhada. No entanto, eles nunca se casaram oficialmente. Maria faleceu recentemente, deixando João como único sobrevivente.\n\nJoão está muito triste e confuso sobre o que fazer com a casa e os bens que eles compartilhavam. Ele não sabe se tem direito à herança de Maria e como isso funcionaria.\n\nProblema:\nJoão precisa saber se ele tem direito à herança de Maria e como isso funcionaria, considerando que eles estavam em uma união estável e não eram casados oficialmente.\n\nJustificativa:\nEssa história se encaixa nos artigos 1.784 a 1.790 do Código Civil Brasileiro, que tratam da sucessão em geral e da legitimação para suceder. Em particular, o art. 1.790 trata da participação da companheira ou do companheiro na sucessão do outro, quanto aos bens adquiridos onerosamente na vigência da união estável.\n\nNesse caso, João, como companheiro de Maria, tem di

In [54]:
story = response.content.split('\n\nJustificativa')[0].split('\n\n')[1]

In [48]:
from langchain_core.output_parsers import JsonOutputParser
from langchain_core.prompts import PromptTemplate
from langchain_core.pydantic_v1 import BaseModel, Field

# Define your desired data structure.
class Story(BaseModel):
    historia: str = Field(description="story about the article")
    justificativa: str = Field(description="answer why the story is related to articles")

# Set up a parser + inject instructions into the prompt template.
parser = JsonOutputParser(pydantic_object=Story)

prompt = PromptTemplate(
    template=sys_story_message + "\n{format_instructions}\n{query}\n",
    input_variables=["query"],
    partial_variables={"format_instructions": parser.get_format_instructions()},
)

query = user_story_prompt.format(context=context,
                                 articles=topic,
                                 examples=exemplos)
chain = prompt | llama3 | parser

chain.invoke({"query": query})

{'historia': 'Maria e João estavam em uma união estável há 10 anos. Eles compraram uma casa juntos e tinham uma vida financeira compartilhada. Infelizmente, João faleceu recentemente. Maria está agora tentando entender como a herança será dividida. Ela sabe que eles não tinham filhos juntos, mas João tinha um filho de um relacionamento anterior. Maria está preocupada em saber se ela tem direito a uma parte da herança.',
 'justificativa': 'Essa história se relaciona com os artigos 1.790 do Código Civil Brasileiro, que trata da sucessão em união estável. De acordo com o artigo, Maria, como companheira de João, tem direito a uma quota da herança. Além disso, como João tinha um filho de um relacionamento anterior, Maria concorrerá com ele na herança. O artigo 1.790 estabelece as condições para a divisão da herança nesse caso.'}

In [55]:
question_prompt = """
Consider:
- a given context about a part of the Código Civil Brasileiro.
- some articles in this part.
- some real example questions
- and a real world story

Create a question that could be answer by one or more articles in context.
The question should be answered by the articles.

Also justify your question showing possible answer in articles.
Answer in Brazilian Portuguese.
"""

user_question_prompt = """
- "context": "{context}",
- "articles": "{articles}",
- "story": "{story}"
""".strip()

# Define your desired data structure.
class Question(BaseModel):
    pergunta: str = Field(description="question that could be answered by articles")
    resposta_justificada: str = Field(description="part of articles that answer the question")

# Set up a parser + inject instructions into the prompt template.
parser = JsonOutputParser(pydantic_object=Question)

prompt = PromptTemplate(
    template=question_prompt + "\n{format_instructions}\n{query}\n",
    input_variables=["query"],
    partial_variables={"format_instructions": parser.get_format_instructions()},
)

query = user_question_prompt.format(context=context,
                                 articles=topic,
                                 story=story)
chain = prompt | llama3 | parser

chain.invoke({"query": query})

{'pergunta': 'Quais são os direitos de João na herança de Maria, considerando que eles estavam em uma união estável e não tinham filhos comuns?',
 'resposta_justificada': 'De acordo com o Art. 1.790 do Código Civil Brasileiro, João, como companheiro de Maria, participará da sucessão da herança, tendo direito a uma quota equivalente à que por lei for atribuída ao filho, se concorrer com filhos comuns, ou à metade do que couber a cada um daqueles, se concorrer com descendentes só do autor da herança.'}

https://leopoldonascimento.adv.br/heranca-10-duvidas-respondidas/

1. Para destinar a herança é preciso ter testamento?
A escolha de um método para transferência de patrimônio de uma pessoa após a sua morte é realizada através de um planejamento sucessório. Para isso, existem diferentes alternativas.

Entre os modos mais conhecidos, temos o testamento.

O testamento permite a divisão dos bens, desde que os limites legais da herança legítima sejam respeitados.

Mas há sim outras opções.

Entre elas, temos doação, seguro, fundos de investimentos, holding familiar e previdência privada, por exemplo.

2. Posso deixar meus bens para quem eu quiser?
A lei brasileira estabelece que a herança pode ser deixada para qualquer pessoa, mas não em sua totalidade.

Isso porque 50% dos bens pertencem aos herdeiros necessários (abaixo, no item 3, explicamos melhor sobre eles).

Sendo assim, apenas os outros 50% podem ser transferidos conforme a preferência.

Por exemplo, considere um casal com o regime de comunhão total de bens que possuem filhos.

No caso de terem um imóvel no valor de R$ 500 mil, eles poderiam transmitir 50% desse bem (R$ 250 mil) para qualquer pessoa, e o restante seria reservado obrigatoriamente para os seus herdeiros, no caso, os filhos.

3. Qual a diferença entre herdeiros necessários e herdeiros testamentários?
De acordo com o Código Civil, existem dois tipos de herdeiros:

Herdeiros necessários ou legítimos: são os descendentes (filhos, netos etc.), os ascendentes (pais, avós, bisavós etc.) e o cônjuge sobrevivente.

Herdeiros testamentários: como o próprio nome sugere, são aqueles beneficiados através de disposição testamentária pela pessoa que faleceu.

Temos um artigo completo sobre o assunto, clique aqui e saiba mais!

4. Posso dispor da minha herança em vida?
A herança é transmitida aos herdeiros após o falecimento de uma pessoa. Mas a sua divisão pode ser feita em vida através de um planejamento sucessório.

Muitas pessoas escolhem antecipar essa divisão para evitar conflitos familiares e garantir que as suas vontades sejam respeitadas.

Mas como é possível fazer essa partilha em vida?

Como citamos anteriormente, o planejamento sucessório pode ser realizado através de testamento, holding familiar, doação, contratos etc.

Para que ele seja feito de maneira correta é importante contar com profissionais especialistas no assunto, que te ajudam a definir o melhor método para o seu caso, considerando valores dos bens, tributos, entre outros aspectos fundamentais para uma boa escolha.

5. Filhos fora do casamento podem receber herança?
Perante a lei, não há distinção entre os filhos. Então, sim, todos possuem o mesmo direito como herdeiros, independentemente de sua origem.

6. Enteados são herdeiros obrigatórios?
Os herdeiros necessários estão dispostos no Art. 1.829 do Código Civil, em que a sucessão legítima defere-se na ordem seguinte: descendentes, ascendentes, cônjuge sobrevivente e colaterais.

O estado de filiação se dá de duas maneiras: nascimento ou adoção. O enteado não se encaixa em nenhuma das duas possibilidades.

Portanto, ele terá direito a parte da herança apenas em caso de disposição em testamento.

Há exceção?

Sim, no caso de padrastros e madrastas que podem ser mais presentes que os próprios pais/mães, ocorre a filiação socioafetiva, que gera a possibilidade do enteado se tornar herdeiro legítimo.

A socioafetividade é tratada na Lei n. 8.069, de 13 de julho de 1990, que dispõe sobre o Estatuto da Criança e do Adolescente.

7. Em caso de não haver herdeiros, quem fica com a herança?
No caso de não haver herdeiros necessários, ascendentes, cônjuge ou colaterais, a herança ficará em poder do Estado.

Exceto em casos de nomeação de outros herdeiros em testamento.

8. As dívidas também são herdadas?
As dívidas de uma pessoa falecida deverão ser quitadas com o patrimônio dela.

Ou seja, o herdeiro não é responsável pelos débitos deixados e nunca precisará pagá-los com os seus bens pessoais.

Se esse valor não for quitado pelo próprio patrimônio, as dívidas serão parcialmente pagas e os herdeiros não receberão nada.

Para exemplificar, imagine que uma pessoa falece e deixa uma dívida de R$100 mil, mas o seu patrimônio corresponde a R$ 80 mil.

A dívida será paga parcialmente com o valor do patrimônio (R$ 80 mil), ou seja, os herdeiros não receberão nada, mas não serão responsáveis pelos R$ 20 mill que ficarão em débito.

9. Em caso de união estável o cônjuge sobrevivente tem direito à herança?
A união estável é definida nos termos do Art.1.723 do Código Civil.

Por decisão do Supremo Tribunal Federal (STF), no caso de comprovação da união estável, os companheiros possuem direito a herança.

O Art. 1.790 do Código Civil diz que:

“A companheira ou o companheiro participará da sucessão do outro, quanto aos bens adquiridos onerosamente na vigência da união estável, nas condições seguintes:

I – se concorrer com filhos comuns, terá direito a uma quota equivalente à que por lei for atribuída ao filho;

II – se concorrer com descendentes só do autor da herança, tocar-lhe-á a metade do que couber a cada um daqueles;

III – se concorrer com outros parentes sucessíveis, terá direito a um terço da herança;

IV – não havendo parentes sucessíveis, terá direito à totalidade da herança.”

10. Preciso de um advogado para fazer um testamento?
Oficialmente, não é obrigatório um advogado para criar o documento de testamento. No entanto, desconsiderar um apoio jurídico pode trazer importunações.

Um advogado especialista em Direito Sucessório pode ajudar a garantir que a vontade do testador seja respeitada e que a adequação técnica seja feita corretamente.

Ao optar por não procurar esse profissional, é possível comparecer a um cartório juntamente a duas testemunhas e realizar o testamento.

Mas caso você opte por auxílio ao lidar com esse documento de extrema importância, procure por profissionais de confiança.

1. Qual a diferença do divórcio consensual e o divórcio litigioso?

O divórcio consensual nada mais é do que aquele que é conhecido como “amigável”. Nele, ambos os cônjuges de comum acordo desejam se divorciar, sem que haja brigas quanto a partilha dos bens. Neste caso, poderá ser celebrado até mesmo no cartório, tendo constituído um mesmo advogado. Já o Litigioso é aquele que obrigatoriamente deve ser realizado na justiça. Isso porque, tem-se conflito entre os envolvidos, ou então, existe um menor de idade envolvido, o que faz com que se torne obrigatório que seja realizado judicialmente.

2. Como fica os bens após o divórcio?

Em relação a partilha dos bens após o divórcio, irá depender de qual regime de bens foi escolhido no casamento. O regime de bens escolhido irá interferir diretamente na divisão.

3. Quanto custa para se divorciar?

Não tem como definir um valor único para o divórcio. Quando realizado em cartório, cada um possui uma tabela própria de recolhimentos de taxas e emolumentos. Caso seja judicial, existem os custos do processo. Em ambos os casos é necessário também estar atento aos honorários do advogado atuante na causa.

4. O cônjuge tem direito à pensão após o divórcio?

Apesar de menos com o pedido de pensão pelo cônjuge após o divórcio, é possível sim realizar o pedido. Porém, isso só irá acontecer caso fique demonstrado que não possui condições de trabalhar, seja por ter uma doença, ou seja por questões do próprio casamento (o ex-marido não deixava trabalhar e a esposa dependia dele). Condições da idade também são levadas e consideração pelo juiz no momento de apreciar o pedido.

5. Posso pedir danos morais caso o motivo do divórcio seja traição?

Sim. O cônjuge pode ajuizar um pedido de danos morais em decorrência de uma traição. Porém, não basta a simples alegação da traição. Será necessário comprovar que a traição e o consequente divórcio geram grandes prejuízos tanto emocionais como psicológicos. A traição, por si só, não é motivo para dar direito a indenização.

6. Como reconhecer meu divórcio no Brasil quando o divórcio foi feito em outro país?

Existem duas formas de se reconhecer o divórcio estrangeiro no Brasil. Através do processo de homologação de divórcio estrangeiro perante o STJ (Superior Tribunal de Justiça), caso o divórcio tenha sido litigioso, ou se houver partilha de bens, ou ainda, se o casal tem filhos. Ou então averbação direta no cartório onde foi registrado o casamento. Avalie um advogado para saber qual dessas duas modalidades melhor te atende.

7. Me casei no exterior, então também sou casada no Brasil?

O casamento celebrado no exterior é válido no Brasil, porém, só terá eficácia depois de registrado no país. O registro pode ser feito no Consulado Brasileiro ou diretamente no cartório do 1º Ofício em Brasília, caso não seja residente no exterior.

8. Posso me divorciar no Brasil estando fora do país?

Sim. Mas somente através de uma procuração específica conferindo tal autorização ao advogado que irá te representar. Se o casamento foi no exterior, será necessário primeiro o registro do casamento em cartório brasileiro, para depois a realização do divórcio.

9. Me casei no Brasil, posso me divorciar no exterior?

A resposta dessa pergunta é depende. Isso porque irá depender da lei de cada país. Geralmente é possível. Uma consultoria jurídica especializada irá te orientar melhor sobre isso.

10. Quais são os documentos para homologar meu divórcio no Brasil?

Os documentos para a homologação do divórcio no Brasil são: procuração, declaração de anuência do ex-cônjuge, sentença estrangeira apostilada (seguindo as diretrizes da Convenção de Haia para os países signatários, caso contrário o documento deve ser legalizado no Consulado Brasileiro), além da tradução juramentada da sentença estrangeira.

11. O que é divórcio qualificado e divórcio simples?

Divórcio simples é aquele em que na sentença de divórcio não há nenhuma outra questão disposta a não ser o novo estado civil dos envolvidos. Ou seja, não há nada relacionado a partilha de bens, guarda de filhos, pensão, etc. Já o divórcio qualificado é o contrário, é quando há qualquer um desses pontos sendo definidos na sentença.

12. Posso manter o sobrenome do outro cônjuge após o divórcio?

Sim. O mais comum é a escolha pela retirada do nome, e a volta do nome de solteiro (a). Porém, é possível manter o mesmo sobrenome. Essa será uma escolha pessoal.

13. Posso parar de pagar pensão assim que meu filho completar 18 anos?

Não. Muitos pais que pagam pensão acreditam que assim que o filho completa 18 anos podem parar automaticamente de pagar a pensão. Mas, não é assim que as coisas funcionam. Será necessário o processo judicial para colocar fim a essa obrigação

14. É possível alterar o valor da pensão alimentícia fixada pelo Juiz?

Sim. Através de uma ação que chamamos de revisional de alimentos, pode ser discutido o valor da pensão, seja para aumentar ou diminuir.

15. Tenho mais de 18 anos, mas não consigo me sustentar ainda. Posso pedir pensão alimentícia para os meus pais?

Sim. O objetivo da pensão alimentícia é o de prestar alimentos à pessoa que não tem condições de prover seu próprio sustento. Apenas a maioridade não significa que uma pessoa consegue se sustentar. Principalmente quando o jovem está cursando a faculdade. Comprovada a necessidade o juiz irá estender o período de pagamento da pensão.

16. Meu filho vai completar 18 anos, ele vai deixar de receber a pensão alimentícia?

Não. Como respondido anteriormente, o simples fato de o filho completar 18 anos não dá direito ao alimentante de que pare de pagar a pensão. É necessário propositura de uma ação para que esse prazo seja ampliado de acordo com a nova realidade do filho. Muitas vezes a pensão é extremamente necessária para custear custos com estudo, moradia, alimentação, etc.

17. Posso alterar o regime de bens que escolhi para o meu casamento para qualquer outro, ou há alguma restrição?

Sim. Mas para que isso seja possível, você não pode ter se casado pelo regime da separação obrigatória de bens (maiores de 70 anos). De acordo com o artigo 1639 do Código Civil Brasileiro, a modificação do regime de bens depende dos seguintes requisitos: i) pedido formulado por ambos os cônjuges ii) autorização judicial; iii) indicação de motivo relevante; iv) inexistência de prejuízo de terceiros e dos próprios cônjuges.

18. Como solicitar revisão da pensão alimentícia?

A revisão deve ser solicitada judicialmente, através de uma ação revisional de alimentos. Porém, deve ser justificado o pedido de revisão, demonstrando que a situação de quem recebe ou de quem está pagando a pensão mudou, tornando necessário o ajuste do valor.

19. Como funciona uma audiência de pensão alimentícia?

Nesse tipo de situação presa-se pela conciliação. Dessa forma, as partes terão a oportunidade de conversar, e buscar conciliar. Dependendo da fase em que estiver o processo, podem ser apresentadas provas, documentos, ou ainda, ouvida testemunhas.

20. Quanto tempo demora um processo de pensão alimentícia?

Não é possível estimar o tempo que leva esse tipo de processo, já que depende de diversos fatores (lugar em que tramita o processo, presença de todos os documentos, consenso ou litígio entre os envolvidos, etc.).

E aí, ficou com alguma dúvida? Nosso objetivo é o de simplificar questões que costumam causar debates e dúvidas a quem se depara com elas em seu dia a dia. Se você precisa de um esclarecimento mais específico ao seu caso, entre em contato. Nossa equipe pode te atender e instruir da melhor forma para resolver o seu problema.



In [30]:
data['parte especial']['LIVRO V'].keys()

dict_keys(['nome', 'TÍTULO I', 'TÍTULO II', 'TÍTULO III', 'TÍTULO IV'])